stable/verdaccio/README.md

```diff
@@ -0,0 +1,125 @@
+# Verdaccio
+
+[Verdaccio](http://www.verdaccio.org) is a lightweight private
+[NPM](https://www.npmjs.com) proxy registry.
+
+## TL;DR;
+
+```
+$ helm install stable/verdaccio
+```
+
+## Introduction
+
+This chart bootstraps a [Verdaccio](https://github.com/verdaccio/verdaccio)
+deployment on a [Kubernetes](http://kubernetes.io) cluster using the
+[Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
```

Comment:
We support the current and the previous minor version. So, 1.7+ would be correct.